release-notes/6.0/preview/api-diff/preview1/Asp.Net/6.0-preview1_Microsoft.AspNetCore.Http.Features.md

```diff
@@ -0,0 +1,19 @@
+# Microsoft.AspNetCore.Http.Features
+
+``` diff
+ namespace Microsoft.AspNetCore.Http.Features {
+-    public interface IHttpBufferingFeature {
```

Comment:
@Tratcher did we remove these? Won't this break any middleware that was compiled for 2.x?

Reply:
Yes, It has been obsolete and non-functional since 3.0.